packages/models.py

```diff
@@ -0,0 +1,163 @@
+import enum
+import uuid as uuid
+from django.db import models
+
+
+class City(models.Model):
+    CENTER = 'Central'
+    NORTH = 'Northern'
+    SOUTH = 'Southern'
+
+    AREA_CHOICES = [
+        (CENTER, 'Central'),
+        (NORTH, 'Northern'),
+        (SOUTH, 'Southern')
+    ]
+
+    name = models.CharField(max_length=64, unique=True, primary_key=True)
+    area = models.CharField(max_length=16, choices=AREA_CHOICES)
+
+    def __str__(self):
+        return self.name
+
+
+class Street(models.Model):

```

Comment:
Why separate everything into different models? It's pretty weird. I understand it makes it more comfortable for the why you work in the frontend, but it makes less sense in the way addresses work. A street is not independent, it is part of a full address the includes a city.

After all, you can have a street `banana` in one city that doesn't exist in the another.

Reply:
It was created only to simplify the street autocomplete, it doesn't harm performance or anything. So I'd say it's worth it. Would you?

Reply:
If you change the way autocomplete works, then this is might be problematic. But change it when that happens.